src/pykeen/models/unimodal/kg2e.py

```diff
@@ -126,17 +125,38 @@ def _kullback_leibler_similarity(
 
 
 class KG2E(EntityRelationEmbeddingModel):
-    """An implementation of KG2E from [he2015]_.
+    r"""An implementation of KG2E from [he2015]_.
+
+    KG2E aims to explicitly model (un)certainties in entities and relations (e.g. influenced by the number of triples
+    observed for these entities and relations). Therefore, entities and relations are represented by probability
+    distributions, in particular by multi-variate Gaussian distributions $\mathcal{N}_i(\mu_i,\Sigma_i)$
+    where the mean $\mu_i \in \mathbb{R}^d$ denotes the position in the vector space and the diagonal variance
+    $\Sigma_i \in \mathbb{R}^{d \times d}$ models the uncertainty.
+    Inspired by the :class:`pykeen.models.TransE` model, relations are modeled as transformations from head to tail
+    entities: $\mathcal{H} - \mathcal{T} \approx \mathcal{R}$ where
+    $\mathcal{H} \sim \mathcal{N}_h(\mu_h,\Sigma_h)$, $\mathcal{H} \sim \mathcal{N}_t(\mu_t,\Sigma_t)$,
+    $\mathcal{R} \sim \mathcal{P}_r = \mathcal{N}_r(\mu_r,\Sigma_r)$, and
+    $\mathcal{H} - \mathcal{T} \sim \mathcal{P}_e = \mathcal{N}_{h-t}(\mu_h - \mu_t,\Sigma_h + \Sigma_t)$
+    (since head and tail entities are considered to be independent with regards to the relations).
+    The interaction model measures the similarity between $\mathcal{P}_e$ and $\mathcal{P}_r$ by
+    means of the KL divergence:
```

Comment:
The sentence is strange, since we first say it measures the similarity as follows ... And then say we can also chose a different variant. It would be better understandable if we directly say that we can choose from those two similarity measures.

Reply:
This is how it was written in the paper (with slight modifications). Please take care of updating it there first, then later we can make another PR to update it here too